spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'restaurant'
+require 'menu'
+require 'text'
+
+describe 'User Stories' do
+
+  let(:menu) {Menu.new}
+  let(:restaurant) {Restaurant.new}
+
+  before do
+    restaurant.add_item("Bread", 2)
+    restaurant.add_item("Chicken Tagine", 4)
+  end
+
+  it 'Displays the menu' do
+    expect(restaurant.display_menu).to eq menu.menu_list
+  end
+
+  it 'allows a user to add items' do
+    restaurant.add_item("Bread")
+    expect(restaurant.order).to include "Bread" => 1
+  end
+
+  it 'allows a user to add multiple items' do
+    expect(restaurant.order).to include "Chicken Tagine" => 4
+  end
+
+  it 'raises an error when an item is not on the menu' do
+    expect{restaurant.add_item("Cheese")}.to raise_error 'No such item on the menu'
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)